packages/repository-tests/src/crud/transactions.suite.ts

```diff
@@ -140,13 +162,19 @@ export function transactionSuite(
         it('should not use transaction with another repository', async () => {
           const ds2Options = Object.assign({}, dataSourceOptions);
           ds2Options.name = 'anotherDataSource';
-          ds2Options.database = ds2Options.database + '-new';
+          ds2Options.database = ds2Options.database + '_new';
+
+          // need to manually create the database for PostgreSQL otherwise it
+          // complains that database doesn't exist
+          if (ds.connector && ds.connector.name === 'postgresql') {
+            await ds.execute(`DROP DATABASE IF EXISTS ${ds2Options.database};`);
+            await ds.execute(`CREATE DATABASE ${ds2Options.database};`);
+          }
```

Comment:
I believe it's because it's using the same model. If we create a new repository class with the same datasource and migrate it, it'll use the same table for both repositories, from my understanding.

That being said, I'll move the database setup to the travis setup.

Reply:
I still find this test confusing, but I guess it's not the job of this pull request to fix that. I like that you moved the database setup to Travis CI build steps, I think this thread can be considered as resolved 👍